javascript/javascript1/week1/smart-ease io/Ez-Namey.js

```diff
@@ -0,0 +1,5 @@
+firstWord = ['WestCoast', 'Smooth', 'Network', 'servo', 'New', 'Heavenly', 'Giant', 'Brighter', 'Pioneer', 'AllPro']
```

Comment:
any specific reason to keep 'firstWod' and 'secondWords' without any declaration key 'like var, let or const'

Reply:
oh no, actually i forgot the declaration